client/vite.config.js

```diff
@@ -3,11 +3,14 @@ import react from "@vitejs/plugin-react";
 
 // https://vitejs.dev/config/
 export default defineConfig({
+  build: {
+    outDir: "./build",
+  },
   plugins: [react()],
   server: {
     port: 3000,
     proxy: {
-      "/api": "http://127.0.0.1:8000",
+      "/api": "https://mythicdb.onrender.com",
```

Comment:
proxy is only used in development, url should be reverted back to the development environment